main.js

```diff
@@ -8,6 +8,97 @@ const About = document.querySelector('.about-me');
 const form = document.querySelector('.form');
 const footer = document.querySelector('.footer');
 const trigger = document.getElementById('btn-trigger');
+const hideScrollBar = document.querySelector('body');
+const worksData = {
```

Comment:
- [ ] Kindly use an array to store the project's data. This will make it easier to update your portfolio with new projects.